website/payments/admin.py

```diff
@@ -531,12 +531,147 @@ def export_csv(self, request: HttpRequest, queryset: QuerySet) -> HttpResponse:
     export_csv.short_description = _("Export")
 
 
+class BankAccountInline(admin.TabularInline):
+    model = BankAccount
+    fields = (
+        "iban",
+        "bic",
+        "mandate_no",
+        "valid_from",
+        "valid_until",
+        "last_used",
+    )
+    show_change_link = True
+
+    can_delete = False
+
+    def has_add_permission(self, request, obj=None):
+        return False
+
+    def has_change_permission(self, request, obj=None):
+        return False
+
+
+class PaymentInline(admin.TabularInline):
+    model = Payment
+    fields = (
+        "created_at",
+        "type",
+        "amount",
+        "topic",
+        "notes",
+        "batch",
+    )
+
+    show_change_link = True
+
+    can_delete = False
+
+    def has_add_permission(self, request, obj=None):
+        return False
+
+    def has_change_permission(self, request, obj=None):
+        return False
+
+
+class ThaliaPayEnabledFilter(admin.SimpleListFilter):
+    title = "Thalia Pay enabled"
+    parameter_name = "tpay_enabled"
+
+    def lookups(self, request, model_admin):
+        return ("Yes", "Yes"), ("No", "No")
```

Comment:
Why use these capitalised words as filter values? Wouldn't it make more sense to use integers or plain booleans? Something that is at least lowercased like for the other filter.